src/app/content/highlights/components/HighlightStyles.tsx

```diff
@@ -32,6 +33,33 @@ export const stickyNoteMeasures = {
   width: 29.8, /* to allow text to fit in one line with tooltip */
 };
 
+const printStyles = {
```

Comment:
please just put these inline so you don't have to keep going up/down to tell what anything does